pkg/api/constants/consts.go

```diff
@@ -20,3 +20,28 @@ const (
 	APIKeysPrefix                = "zak_"
 	CallbackUIQueryParam         = "callback_ui"
 )
+
+var (
+	openIDSupportedProviders = [...]string{"google", "gitlab", "dex"} //nolint: gochecknoglobals
+	oauth2SupportedProviders = [...]string{"github"}                  //nolint: gochecknoglobals
+)
+
+func IsOpenIDSupported(provider string) bool {
```

Comment:
Maybe these methods should move out of the constants package? They are not constants, maybe move back to pkg/api?